### PR TITLE
updated-documentation

### PR DESCRIPTION
**Pull request by arcoprova**
id-page-image1-before
id-page-image1-now
id-page-image2-before
id-page-image2-now
id-page-image3-before
id-page-image3-now
id-page-upper-before
id-page-upper-now
fixed horizontal scrollbar issue on ftd.dev/id page.
input-page-before
input-page-now
updated documentation on  ftd.dev/input page. 
text-page-output1-before
text-page-output1-now
text-page-output2-before
text-page-output2-now
text-page-output3-before
text-page-output3-now
text-page-output4-before
text-page-output4-now
updated output text colors on  ftd.dev/text page.

### CODE REVIEW

File: alternatives.ftd [hunk 1]
```diff
@@ -1,4 +1,7 @@
 -- ft.page: Alternatives
+toc: $config.alternatives-toc
+sub-sections: $config.why-header
```
Comment (amitu) [inline alternatives.ftd]: Why did you do this? Remove toc/sub-sections parameters from everywhere.

File: FPM.ftd [hunk 2]
```diff
@@ -5,9 +5,14 @@ zip: github.com/fifthtry/ftd.dev/archive/refs/heads/main.zip
 language: en
 
 -- fpm.dependency: fifthtry.github.io/doc-site as ft
+
+-- fpm.dependency: fifthtry.github.io/brand-switcher as bs
```
Comment (Arpita-Jaiswal) [inline FPM.ftd]: Remove this and also check if it's imported anywhere. `brand-switcher` is now known as `product-switcher`

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: FPM.ftd [hunk 3]
```diff
@@ -5,9 +5,14 @@ zip: github.com/fifthtry/ftd.dev/archive/refs/heads/main.zip
 language: en
 
 -- fpm.dependency: fifthtry.github.io/doc-site as ft
+
+-- fpm.dependency: fifthtry.github.io/product-switcher as bs
+
 -- fpm.dependency: fifthtry.github.io/product-switcher as ps
```
Comment (Arpita-Jaiswal) [inline FPM.ftd]:
```suggestion
-- fpm.dependency: fifthtry.github.io/product-switcher as ps
```

Don't add dependencies twice. Remove the first one. and try to build it locally.

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: FPM/config.ftd [hunk 4]
```diff
@@ -8,6 +8,10 @@
 
 
 
+-- fpm.sitemap-data sitemap:
+$processor$: sitemap
+
```
Comment (Arpita-Jaiswal) [inline FPM/config.ftd]: Why have we added this?

File: examples/comic.ftd [hunk 5]
```diff
@@ -4,11 +4,16 @@
 -- ftd.color cadetblue: cadetblue
 dark: cadetblue
 
+
 -- ftd.color white: white
 dark: white
 
 -- ft.page: Once upon a time...
 
+-- ft.page: Once upon a time...
```
Comment (Arpita-Jaiswal) [inline examples/comic.ftd]: Why twice?

File: id.ftd [hunk 6]
```diff
@@ -4,6 +4,13 @@
 -- ft.page: ID in `ftd`
 
 
+
+-- boolean is-mobile: false
+
+-- is-mobile: true
+if: $ftd.device == mobile
+
```
Comment (Arpita-Jaiswal) [inline id.ftd]: Are we using it anywhere?

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

**Pull request closed without merging**